scripts/insolard/launch.sh

```diff
@@ -0,0 +1,118 @@
+set -e
+
+GINSIDER_DIR=logicrunner/goplugin/ginsider-cli
+GINSIDER_BIN=$GINSIDER_DIR/ginsider-cli
+TEST_DATA=testdata
+INSOLARD=$TEST_DATA/functional/insolard
+CONTRACT_STORAGE=contractstorage
+
+stop_listening()
+{
+    ports="7777 19191 18181"
```

Comment:
you should't check 7777 since your goplugin use port 18181
its cool to check 18182 (you set mainlisten of goplugin to it)